tests/models/test_openai.py

```diff
@@ -94,13 +94,20 @@
 ]
 
 
-def test_init():
+def test_chat_model_init():
     m = OpenAIChatModel('gpt-4o', provider=OpenAIProvider(api_key='foobar'))
     assert m.base_url == 'https://api.openai.com/v1/'
     assert m.client.api_key == 'foobar'
     assert m.model_name == 'gpt-4o'
 
 
+def test_response_model_init():
```

Comment:
Please move this to `test_openai_responses.py` and we can rename the original test back!